src-electron/ui/ui-util.ts

```diff
@@ -118,9 +118,22 @@ function openFileDialogAndReportResult(
   p.defaultPath = options.defaultPath
   dialog.showOpenDialog(browserWindow, p).then((result) => {
     if (!result.canceled) {
+      let paths : string[] = [];
+      // check if application is running on Windows standalone mode, then replace the backward slashs with forward slash
+      // otherwise it throw an JSON syntax error
```

Comment:
`otherwise it throw an JSON syntax error` to `otherwise it will throw a JSON syntax error`